labs/lab4/chapter4.md

```diff
@@ -30,14 +30,14 @@ docker port wordpress
 
 and taking look at the "PORTS" column for the wordpress site. You 
 can also get your ip address by looking at the address for the 
-eth1 interface after you execute:
+eth0 interface after you execute:
 
 ```bash
-ip addr show dev eth1
+minishift ssh "ip addr show dev eth0"
```

Comment:
i wonder if we should use `minishift ip` here. in our case minishift actually has two ip addresses. one of them is some random .42 address and the other is the one that corresponds to cdk.example.com.